lib/pages/login_page.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/material.dart';
+import 'package:vama_mobile/components/my_button.dart';
+import 'package:vama_mobile/components/my_textfield.dart';
+import 'package:vama_mobile/components/my_header.dart';
+
+class LoginPage extends StatelessWidget{
+   LoginPage ({super.key});
+
+  //text editing controllers
+  final usernameController = TextEditingController();
+  final passwordController = TextEditingController();
+
+  //sign user in method
+   void loginUserIn(){}
```

Comment:
Since we don't currently have proper issues, this method can remain a placholder, but in the future look for the implemntation of request methods on [vama-api-doc](https://vama-api-doc.apidog.io/login-14963081e0).

It is a good practice to throw an "Unimplemnted Exception" in unimplemented methods

Also, you could simplify the name of this method, by calling it "login".

```suggestion
  void loginUserIn() {
    throw UnimplementedError();
  }
```